### PR TITLE
Feature/changing blinds on off to open close commands

### PR DESCRIPTION
**Pull request by CyrilB1531**
Changing On / Off Command to Open / Close for blinds

### CODE REVIEW

**Comment by kiddigital** [comment]
@CyrilB1531 , does this PR make #5285 obsolete?

**Comment by CyrilB1531** [comment]
@kiddigital: Yes it does, I've closed it

File: main/mainworker.cpp [hunk 1]
```diff
@@ -12307,12 +12331,13 @@ bool MainWorker::SwitchLightInt(const std::vector<std::string>& sd, std::string
...
 			lcmd.RADIATOR1.subtype = sTypeSmartwaresSwitchRadiator;
-			PushAndWaitRxMessage(m_hardwaredevices[hindex], (const uint8_t *)&lcmd, nullptr, -1, User.c_str());
+			PushAndWaitRxMessage(m_hardwaredevices[hindex], (const uint8_t*)&lcmd, nullptr, -1, User.c_str());
 		}
 		return true;
+	}
 	case pTypeGeneralSwitch:
 	{
-
+		tRBUF lcmd;
```
Comment (gizmocuz) [inline main/mainworker.cpp:12340]: I don't think this variable declaration is needed?

**Comment by gizmocuz** [comment]
Thank you for this PR and help!

Do you know by any change if this also solves the below issue that has appeared ?

https://github.com/domoticz/domoticz/issues/5287

File: main/mainworker.cpp (hunk 1 above)
Comment (CyrilB1531) [reply main/mainworker.cpp:12340]: It was because I've added braces in case above

**Comment by CyrilB1531** [comment]
> Thank you for this PR and help!
> 
> Do you know by any change if this also solves the below issue that has appeared ?
> 
> #5287

I will test it with zigbee2MQTT to be sure

**Comment by gizmocuz** [comment]
Thank you for the PR!

**Comment by gizmocuz** [comment]
I received some positive feedback already with this PR!

**Comment by CyrilB1531** [comment]
@gizmocuz I have to change domoticz-Zigbee2MQTT python plugin to make it works perfectly with this fix...

**Pull request merged**